src/App.tsx

```diff
@@ -1,24 +1,204 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+// eslint-disable-next-line import/no-cycle
+import { Footer } from './components/Footer/Footer';
+import { Notification } from './components/Notification/Notification';
+
+import { Todo } from './types/Todo';
+
+import {
+  getTodos, addTodo, onDelete, onUpdate,
+} from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 6476;
+
+export enum SortType {
+  ALL = 'All',
+  ACTIVE = 'Active',
+  COMPLETED = 'Completed',
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState(SortType.ALL);
+  const [inputDisable, setInputDisable] = useState(false);
+  const [title, setTitle] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const visibleTodos = useMemo(() => {
+    return (todos.filter((todo) => {
+      switch (filterBy) {
+        case SortType.ALL:
+          return todos;
```

Comment:
filter return only Boolean value, you can't use `todos` here
Also you can remove it, because with All you want to return true every time( default case doing it)
so when you remove this case => you'll go to default and it's ok for you

Reply:
fix it